bolt12/format-string-test.json

```diff
@@ -0,0 +1,47 @@
+[
+    {
+	"comment": "A complete string is valid",
```

Comment:
In regards to my UPPERCASE for QR Codes and lowercase for text presentation, we could duplicate this test to an UPPERCASE version.  

```
{
	"comment": "A complete UPPERCASE string is valid",
	"valid": true,
	"string": "LNO1PQPS7SJQPGTYZM3QV4UXZMTSD3JJQER9WD3HY6TSW35K7MSJZFPY7NZ5YQCNYGRFDEJ82UM5WF5K2UCKYYPWA3EYT44H6TXTXQUQH7LZ5DJGE4AFGFJN7K4RGRKUAG0JSD5XVXG"
    },
```

Reply:
Great minds think alike, I've added two UPPERCASE examples.